openslides/core/static/js/core/core.js

```diff
@@ -605,6 +605,46 @@ angular.module('OpenSlidesApp.core.site', [
             }
         );
     };
+
+    // *** countdown ***
+    $scope.countdowns = [];
+    $scope.messages = [];
+    _.forEach(Projector.get(1).elements, function(element) {
+        if (element.name == 'core/countdown') {
+            $scope.countdowns.push(element);
+        }
```

Comment:
@normanjaeckel I need the countdown uuid within of each element so that I can push the whole element to may array 'countdowns'. Later in template I want to access the id by countdown.id. Can you change it?